core/contracts/oracle/implementation/Constants.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.6.0;
+
+
+/**
+ * @title Stores common interface names used through the protocol through registration in the Finder.
```

Comment:
nit: typo I think it should be "throughout" and probably change "protocol" to "DVM"